src/pages/experiments/[experimentId]/data-exploration/components/cell-sets-tool/CellSetsTool.jsx

```diff
@@ -72,29 +102,92 @@ const CellSetsTool = (props) => {
       );
     }
 
+    let operations = null;
+    const numSelected = getNumberOfCellsSelected();
+
+    if (numSelected) {
+      operations = (
+        <Space>
+          <Text type='secondary'>
+            {numSelected}
+            {' '}
+            cell
+            {numSelected === 1 ? '' : 's'}
+            {' '}
+            selected
+          </Text>
+          {/* <Tooltip title='Create complement of selected cells'>
+            <Button type='dashed' icon={<BlockOutlined />} size='small' />
+          </Tooltip>
+
+          <Tooltip title='Create intersection of selected cells'>
+            <Button type='dashed' icon={<SplitCellsOutlined />} size='small' />
+          </Tooltip>
+
+          <Tooltip title='Combine selected cells'>
+            <Button type='dashed' icon={<MergeCellsOutlined />} size='small' />
+          </Tooltip> */}
+        </Space>
+      );
+    }
+
+    const recluster = () => {
+      dispatch(resetCellSets(experimentId));
+    };
+
+    const cellSetTreeData = composeTree(hierarchy, properties, 'cellSet');
+    const metadataTreeData = composeTree(hierarchy, properties, 'metadataCategorical');
```

Comment:
I don't see where and how the third argument of that function is getting handled